microsoft/botframework-solutions
# [BotBuilder Skills][TypeScript] Implement tests

## Description
- Implement `SkillEvents` instead of `Events` in `skillDialog` file
- Implement `skillDialogInvocationTest`
- Implement `skillDialogSlotFillingTest`
- Implement `manifestTest`
- Implement `skillMiddlewareTest`
- Implement **mock classes**
- Add **helpers**
- Change **test folder structure**
- Update `npm-shrinkwrap.json`

## Testing Steps

1. Go to `botframework-solutions\lib\typescript\`
1. Run `rush update`
1. Run `run rebuild`
1. Go to `botframework-solutions\lib\typescript\botbuilder-skills`
1. Run `npm run test` and check the following output

![image](https://user-images.githubusercontent.com/11904023/58173642-43602080-7c72-11e9-9714-075b669350f6.png)

## Checklist
